slsDetectorSoftware/src/slsDetector.cpp

```diff
@@ -607,6 +607,23 @@ void slsDetector::setQuad(const bool enable) {
     }
 }
 
+void slsDetector::setReadNLines(const int value) {
+    FILE_LOG(logDEBUG1) << "Setting read n lines to " << value;
+    sendToDetector(F_SET_READ_N_LINES, value, nullptr);
+    FILE_LOG(logDEBUG1) << "Setting read n lines to " << value << " in Receiver";
+    if (shm()->useReceiverFlag) {
+        sendToReceiver(F_SET_RECEIVER_READ_N_LINES, value, nullptr);
+}
+}
+
+int slsDetector::getReadNLines() {
```

Comment:
Philosophical question: Do we need both log lines? In this function we have two lines of logging and three lines of actual code... maybe it's possible to move the logging into sendToDetector?